src/app/helpers/x-pub.ts

```diff
@@ -0,0 +1,104 @@
+"use server"
+import { auth } from "@/auth"
+import endpoints from "@/config/endpoints"
+import { xpubInputProp } from "@/types/profile/index"
+
+export const addXpub = async (payload: xpubInputProp) => {
+	const session = await auth()
+	if (!session) {
+		return {
```

Comment:
Can we have a function 'response(message, status, success)' and return it here. We can reuse the function and pass values